overlord/devicestate/export_test.go

```diff
@@ -164,6 +164,14 @@ func RemodelDeviceBackend(remodCtx remodelContext) storecontext.DeviceBackend {
 	}).deviceBackend()
 }
 
+func (mgr *DeviceManager) SetManagerSystemMode(mode string) (restore func()) {
```

Comment:
we already have SetSystemMode, also unsure why we would need a restore, we make a new set of managers per test I think

Reply:
well what do you know we do have one of those don't we, I didn't see that before